extensions/amp-consent/0.1/amp-consent.css

```diff
@@ -81,7 +81,13 @@ iframe.i-amphtml-consent-ui-fill {
   border: none;
 }
 
+/**
+ * CMP Modal class appolied when asking for CMP consent.
+ * See: https://developers.google.com/web/updates/2016/12/url-bar-resizing
+ * For the reasoning of mixing percentages with viewport units
+ */
 amp-consent.i-amphtml-consent-ui-iframe-active {
+  position: fixed !important;
```

Comment:
Discussed offline. This should be safe to remove because we've already applied `position: fixed` to amp-consent.